src/helpers/__tests__/plist.test.ts

```diff
@@ -13,6 +16,19 @@ describe('plist util', () => {
       expect(plist.getPropertyValue('CFBundleVersion')).toBe(12)
     })
 
+    it('returns the value for an env variable', () => {
+      process.env.EXECUTABLE_NAME = 'executable name'
+      const plist = parsePlist('src/helpers/__tests__/plist-fixtures/Info.plist')
+      expect(plist.getPropertyValue('CFBundleExecutable')).toBe('executable name')
+    })
+
+    it('returns an empty string for an env variable that is not declared', () => {
+      const plist = parsePlist('src/helpers/__tests__/plist-fixtures/Info.plist')
+      expect(() => plist.getPropertyValue('CFBundleExecutable')).toThrowError(
+        "Environment variable EXECUTABLE_NAME for property value $(EXECUTABLE_NAME) wasn't found."
```

Comment:
nit: Wouldn't it be more helpful to say something like this instead?

> Environment variable `$(EXECUTABLE_NAME)` for property `CFBundleExecutable` wasn't found.

IMO, saying both "EXECUTABLE_NAME" and "$(EXECUTABLE_NAME)" in the message is redundant, since you [`return propertyValue`](https://github.com/DataDog/datadog-ci/pull/849/files#diff-7d620898cbcf1cd44bc614f8927b751be084e36d4cb1bf8dd8dd1867e793a497R15) if the regex doesn't have a match.

Reply:
Totally agree, that's much better 👍